Doc/library/os.rst

```diff
@@ -1230,6 +1231,16 @@ or `the MSDN <https://msdn.microsoft.com/en-us/library/z0kc8e3z.aspx>`_ on Windo
    .. versionadded:: 3.7
 
 
+.. data:: RWF_APPEND
+
+   Append data to the end of the file. See the description of the flag 'O_APPEND' in ''os.open()''.
```

Comment:
sorry "Provide a per-write equivalent of the `O_APPEND` in ``os.open()``. This flag effect applies only to the data range written by the system call. The *offset* argument does not affect the write operation; the data is always appended to the end of the file. However, if the *offset* argument is ``-1``, the current file *offset* is updated." I am sure vstinner will want to change this description again :)